deploy-manage/kibana-reporting-configuration.md

```diff
@@ -99,16 +99,15 @@ When security is enabled, you grant users access to {{report-features}} with [{{
         If you have a Basic license, sub-feature privileges are unavailable.
         :::
 
+        :::{note}
+        If the **Reporting** options for application features are unavailable, and the cluster license is higher than Basic, contact your administrator.
+        :::
 
         :::{image} /deploy-manage/images/kibana-kibana-privileges-with-reporting.png
         :alt: {{kib}} privileges with Reporting options, Gold or higher license
         :screenshot:
         :::
```

Comment:
No matter what I tried, I couldn't get the second `{note}` to render under the image.